jmclient/jmclient/cryptoengine.py

```diff
@@ -3,16 +3,17 @@
 
 from binascii import hexlify, unhexlify
 from collections import OrderedDict
+import struct
 
 
 from . import btc
 from .configure import get_network
 
 
-TYPE_P2PKH, TYPE_P2SH_P2WPKH, TYPE_P2WPKH = range(3)
+TYPE_P2PKH, TYPE_P2SH_P2WPKH, TYPE_P2WPKH = [b'\x00', b'\x01', b'\x02']
```

Comment:
This change doesn't make sense. It's just an internal identifier.

Reply:
Confirmed in test that this breaks wallet_tool ("No wallet implementation for type 1" on line 849 of jmclient/wallet_utils.py)